packages/playwright/src/index.ts

```diff
@@ -702,6 +705,30 @@ class ArtifactsRecorder {
 
     await this._screenshotRecorder.persistTemporary();
     await this._pageSnapshotRecorder.persistTemporary();
+
+    if (this._pageSnapshotRecorder.shouldCaptureUponFinish())
+      await this._attachErrorPrompts();
+  }
+
+  private async _attachErrorPrompts() {
+    const file = {
+      path: this._testInfo.file,
+      contents: await fs.promises.readFile(this._testInfo.file, 'utf-8'),
+    };
+    for (const error of this._testInfo.errors) {
+      const metadata = this._testInfo.config.metadata as MetadataWithCommitInfo;
+      const prompt = fixTestPrompt(
+          error,
+          file,
+          metadata.gitDiff,
+          this._pageSnapshot, // TODO: maybe capture snapshot when the error is created, so it's from the right page and right time
+      );
+      this._testInfo.attachments.push({
+        name: 'errorPrompt',
```

Comment:
as agreed offline, `_prompt` and hide it from the reports.

Reply:
done